collision_groups:

  - name: body
    type: voxels
    root_name: base_footprint
    tip_name: head_pan_link
    collision_links:
    - name: arm0_turret
      root: arm0_shoulder_pan_link
    - name: arm1_turret
      root: arm1_shoulder_pan_link
    - name: arm2_turret
      root: arm2_shoulder_pan_link

collision_spheres:

  - {name: sh0, x: 0.13, y: 0.0, z: -0.04, radius: 0.16, priority: 5}

  - {name: ua0, x: 0.18, y: 0.0, z: -0.015, radius: 0.11, priority: 4}
  - {name: ua1, x: 0.26, y: 0.00, z: -0.035, radius: 0.08, priority: 4}
  - {name: ua2, x: 0.3275, y: 0.00, z: -0.035, radius: 0.08, priority: 4}
  - {name: ua3, x: 0.40, y: 0.0, z: -0.0, radius: 0.105, priority: 2}

  - {name: fa0, x: 0.13, y: 0.0, z: 0.005, radius: 0.075, priority: 2}
  - {name: fa1, x: 0.20, y: 0.025, z: -0.01, radius: 0.055, priority: 3}
  - {name: fa2, x: 0.20, y: -0.025, z: -0.01, radius: 0.055, priority: 3}
  - {name: fa3, x: 0.265, y: 0.0275, z: -0.015, radius: 0.05, priority: 3}
  - {name: fa4, x: 0.265, y: -0.0275, z: -0.015, radius: 0.05, priority: 3}
  - {name: fa5, x: 0.32, y: 0.0225, z: -0.005, radius: 0.05, priority: 3}
  - {name: fa6, x: 0.32, y: -0.0225, z: -0.005, radius: 0.05, priority: 3}

  - {name: gr0, x: 0.0725, y: -0.0175, z: 0.00, radius: 0.04, priority: 2}
  - {name: gr1, x: 0.0725, y: 0.0175, z: 0.00, radius: 0.04, priority: 2}
  - {name: gr2, x: 0.0925, y: 0.0, z: 0.00, radius: 0.04, priority: 2}

  - {name: lf0, x: 0.03, y: 0.025, z: 0.00, radius: 0.04, priority: 1}
  - {name: lf1, x: 0.08, y: 0.0175, z: 0.00, radius: 0.035, priority: 1}

  - {name: rf0, x: 0.03, y: -0.025, z: 0.00, radius: 0.04, priority: 1}
  - {name: rf1, x: 0.08, y: -0.0175, z: 0.00, radius: 0.035, priority: 1}

  - {name: rft0, x: 0.015, y: 0.005, z: 0.00, radius: 0.03, priority: 1}
  - {name: rft1, x: 0.035, y: 0.005, z: 0.00, radius: 0.02, priority: 1}

  - {name: lft0, x: 0.015, y: -0.005, z: 0.00, radius: 0.03, priority: 1}
  - {name: lft1, x: 0.035, y: -0.005, z: 0.00, radius: 0.02, priority: 1}

   
